src/images/download.py

```diff
@@ -45,6 +46,27 @@ def __init__(self, msg, *msg_args):
         self.msg_args = msg_args
 
 
+def remove_metadata(img):
+    """Open and Close image to remove metadata with pillow."""
+    if not img.lower().endswith('.svg'):
+        size = os.stat(img).st_size
+        img_pil = Image.open(img)
+        img_pil.save(img)
```

Comment:
Asumo que el close se hace cuando sale de contexto de la función, pero no estaria mal cerrarlo explicitamente o usar un context `with `

Reply:
Got it! aplicando cambio.